### DISCUSSION TITLE
Need rainy, snowy and foggy images

### DISCUSSION THREAD

**Discussion by ubicray** [bdd100k/bdd100k#222] [Q&A] [answered]
I apologize if this is a stupid one, but couldn't find any similar questions and help would be greatly appreciated

I need the annotated images from 100k that are tagged with either rain, fog, or snowy weather
Are there any scripts that would help with separating a subset with those images?
Also will need it in coco format, but I believe I can use the to_coco script for that

Thanks for your time!

**Comment by thomasehuang** [comment, marked as answer]
Each frame has an `attributes` field which contains the annotation for the weather. We don't have any scripts to separate it for you, but you should be able to just iterate through the frames and save those that are of interest.
